src/renderer/components/Preference/Advanced.vue

```diff
@@ -208,10 +210,12 @@
       btTracker: convertCommaToLine(btTracker),
       hideAppMenu,
       lastCheckUpdateTime,
+      protocols: {
+        ...protocols
+      },
```

Comment:
Isn't this the same as using only `protocols`? Or did you use this only for code readability?

Reply:
Haha, in order to avoid the following errors.
> Do not mutate vuex store state outside mutation handlers.

Reply:
Yes, I used `redux` before which is similar in its rules. Mutation can lead to some messy bugs.